src/test/ui/async-await/async-unsafe-fn-call-in-safe.rs

```diff
@@ -0,0 +1,16 @@
+// edition:2018
+
+#![feature(async_await, async_closure)]
```

Comment:
```suggestion
#![feature(async_await)]
```
Since you don't use any async closures here :)